orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/utils/HealthHelper.groovy

```diff
@@ -82,6 +82,21 @@ class HealthHelper {
     } as Map
   }
 
+  static boolean someAreDown(Map instance, Collection<String> interestingHealthProviderNames) {
+    List<Map> healths = filterHealths(instance, interestingHealthProviderNames)
+
+    if (!interestingHealthProviderNames && !healths) {
```

Comment:
That does still seem to be a thing (see hasSucceeded implementations in subclasses of AbstractInstancesCheckTask)